Jenkinsfile

```diff
@@ -142,18 +142,13 @@ String unit_packages() {
         need_qb = true
     }
     if (stage_info['target'] == 'centos7') {
-        String packages =  'gotestsum openmpi3 ' +
```

Comment:
Why was gotestsum removed? This is an advanced test runner for the go tests.

Reply:
You're correct, this is needed.  I thought it was odd.

Looking at https://build.hpdd.intel.com/blue/rest/organizations/jenkins/pipelines/daos-stack/pipelines/daos/branches/master/runs/3591/nodes/689/steps/803/log/?start=0 it shows

[2021-01-21T17:55:50.102Z] + dnf -y erase gotestsum openmpi3 hwloc-devel argobots fuse3-libs fuse3 boost-devel libisa-l-devel libpmem libpmemobj protobuf-c spdk-devel libfabric-devel pmix numactl-devel libipmctl-devel python36-tabulate numactl libyaml-devel valgrind-devel
[2021-01-21T17:55:50.102Z] No match for argument: gotestsum
[2021-01-21T17:55:50.102Z] No match for argument: argobots
[2021-01-21T17:55:50.102Z] No match for argument: libisa-l-devel
[2021-01-21T17:55:50.102Z] No match for argument: libpmem
[2021-01-21T17:55:50.102Z] No match for argument: libpmemobj
[2021-01-21T17:55:50.102Z] No match for argument: protobuf-c
[2021-01-21T17:55:50.102Z] No match for argument: spdk-devel
[2021-01-21T17:55:50.102Z] No match for argument: libfabric-devel
[2021-01-21T17:55:50.102Z] No match for argument: pmix
[2021-01-21T17:55:50.102Z] No match for argument: numactl-devel
[2021-01-21T17:55:50.102Z] No match for argument: numactl
[2021-01-21T17:55:50.102Z] No match for argument: libyaml-devel
[2021-01-21T17:55:50.102Z] No match for argument: valgrind-devel

but I see it makes a second attempt at installing these packages later which I'd missed.  I'll revert.

Reply:
Note that this:
```
> [2021-01-21T17:55:50.102Z] + dnf -y erase gotestsum openmpi3 hwloc-devel argobots fuse3-libs fuse3 boost-devel libisa-l-devel libpmem libpmemobj protobuf-c spdk-devel libfabric-devel pmix numactl-devel libipmctl-devel python36-tabulate numactl libyaml-devel valgrind-devel
```

is an **erase** operation, meant to clean things up on nodes that are not refreshed for every test run.  Since we do start fresh, there is nothing to erase, which is what you are seeing up there in the `No match for argument …` messages.

Reply:
Yes, I realised that afterwards when I was backing this out.